### ISSUE TITLE
Move artificial body parts to a separate mod

### ISSUE THREAD

**Issue by Mythos** [Mythos/RimWorld-Extended-Human-Body-Simulation#16]
IMHO artificial body parts should be moved to a separate mod as it is cluttering this mod a bit too much.
Also there was a request for crafting those body parts.

**Comment by EPSIL0N** [comment]
yeah, fair point. Might end up with a few mods or a mod that is mainly patches depending on what prosthetic / organ mod people are using.

**Comment by Mythos** [comment]
I also found out while testing your fix for Crafted Mutants, that EPOE also adds in craftable artificial body parts.
Probably most body parts which are not related to EHBS were just copied over from EPOE.

**Comment by EPSIL0N** [comment]
Yeah, a lot of copying to clean up as that was like V0.17 or earlier EPoE it was from
